CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v1.0.1 (TBD)
```

Comment:
That should be `v1.1.0` since patch releases are not allowed to add features. See point 6 on <https://semver.org/>.